xapian-bindings/configure.ac

```diff
@@ -131,6 +149,10 @@ dnl CXXFLAGS for the duration of configure.
 save_CXXFLAGS=$CXXFLAGS
 CXXFLAGS="$CXXFLAGS $XAPIAN_CXXFLAGS"
 
+if test yes = "$enable_letor"; then
+    CXXFLAGS+="$XAPIANLETOR_CXXFLAGS"
```

Comment:
`+=` isn't portable shell - use the same idiom as just above to append to a variable instead.

Reply:
Also this lack a space before the appended value, so would result in `CXXFLAGS` with a value like `-I/home/olly/git/xapian/xapian-core/include-I/home/olly/git/xapian/xapian-letor/include`...

Reply:
fixed in a fixup commit.